block/src/qcow/mod.rs

```diff
@@ -471,8 +471,9 @@ impl QcowFile {
                 .read(true)
                 .open(path)
                 .map_err(Error::BackingFileIo)?;
-            let backing_file = crate::create_disk_file(backing_raw_file, direct_io)
-                .map_err(|e| Error::BackingFileOpen(Box::new(e)))?;
+            let backing_file =
+                crate::create_disk_file(backing_raw_file, direct_io, max_nesting_depth)
```

Comment:
Why call this generic function - doesn't the backing file have to be QCOW too?

Reply:
In fact the backing file may not be a qcow file, although it is indeed qcow in most cases.

You can refer to: https://github.com/qemu/qemu/blob/v9.0.0/qemu-img-cmds.hx#L56-L60

There are some params: `[-f FMT] [-b BACKING_FILE [-F BACKING_FMT]]`, where `FMT` is the format of the image you will create and `BACKING_FMT` is the format of the backing file.